README.md

```diff
@@ -1,33 +1,48 @@
+
+### What is TC2 *not*?
+* TC2 is *not* a "pro mod"
+
+While much of the work in TC2 will improve the competitive experience, TC2 does not touch gameplay, balance, or competitive rules.
+
+* A vision for the "good old days" of TF2
+
+TC2 will not feature game design changes, gameplay additions, or almost any form of new content. If you're looking for this kind of content, [Team Fortress 2 Classic](https://tf2classic.com/) is more up your alley.
+
+* The modern game, but with new content/updates
 
-Team Fortress 2, but with a lot of fixes, quality of life improvements and performance optimizations!
+You're looking for [Creators.TF](https://creators.tf/).
 
-## About
+* An alternative, more stable game experience
 
-What is Team Comtress 2? It's a version of Team Fortress 2, based on the recent leak which aims to fix many bugs, performance issues, etc. Imagine it like [mastercomfig](https://mastercomfig.com/) super charged!
+TC2 is based on the 2017 TF2 source code leak, which contained an in-development version of Jungle Inferno. This build is rife with issues and incomplete features, and *is not compatible with* the modern game. It isn't even fully playable on its own.
 
-Obviously, as a leaked build, it's not useful for getting better performance in Casual on its own (you can't use this build to connect to any existing servers), but it can help me a lot if you all can test it, so that I am more confident in sending many of these changes to Valve for them to include in the base game! Please let me know how it works for you!
+## Installation
 
-## Install
+**Windows-only for now** (see [#2](https://github.com/mastercomfig/team-comtress-2/issues/2))
 
-1. [Download](https://github.com/mastercomfig/team-comtress-2/releases/latest) `game_clean.zip` from the latest release.
-2. Copy your current `Team Fortress 2` installation to a new folder.
-3. Extract the ZIP download to this new folder.
-4. Make sure you don't have any configs installed.
+1. Copy your current `Team Fortress 2` installation to a new folder.
+2. In your cloned TF2 folder, delete the `tf/custom` folder, as well as `tf/cfg/autoexec.cfg` and `tf/cfg/config.cfg`.
+3. [Download](https://github.com/mastercomfig/team-comtress-2/releases/latest) `game_clean.zip` from the latest release.
+4. Extract the downloaded ZIP to your cloned TF2 folder, overwriting existing files.
 5. Double click `start_tf2.bat`. Note that you must have Steam running.
 6. Enjoy!
 
 Note: TC2 bundles the stock HUD with [CriticalFlaw's HUD fix](https://github.com/CriticalFlaw/TF2-HUD-Fix) in the custom folder
 
 ## New console commands and launch options
 
-Although configs are not recommended (use video options to customize), there are some new customization variables you can try that haven't been added yet!
+Although configs are *not* recommended (use video options to customize), there *are* some new variables you can try out that haven't been added to the in-game options yet!

```

Comment:
I do not like the " to the in-game options" part.

Reply:
I mentioned this because of the "use video options to customize" comment, and it's clear we have the intention of adding these cvars to the advanced options menu. RFC?

Reply:
> I mentioned this because of the "use video options to customize" comment, and it's clear we have the intention of adding these cvars to the advanced options menu. RFC?
When I read it the first time it was clear and conveyed that the team intends on adding more video options ui later, If that's the case this is a good way to put it, imo